hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineModule.java

```diff
@@ -643,12 +643,18 @@ private DecoratedSamlResponseToIdaResponseIssuedByIdpTransformer getResponseToIn
     }
 
     @Provides
-    @Named("AES256DecrypterWithGCM")
+    @Named("EidasAssertionDecrypter")
     private AssertionDecrypter getAES256WithGCMAssertionDecrypter(IdaKeyStore keyStore) {
```

Comment:
Might want to rename this method.

Reply:
Good spot. Will make this change to bring in inline with the @Named name.